packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -670,39 +484,8 @@
 
 // postcss-bem-linter: end
 
-// postcss-bem-linter: define text-field-helper-text
-
-.mdc-text-field-helper-text {
```

Comment:
moved this to `helper-text/mdc-text-field-helper-text.scss`

Reply:
I think we agreed that any CSS which requires a parent CSS class should not be moved into a subelement directory, since it's more important to keep all subelements in sync within any given parent state than it is to keep a subelement in sync with itself within all parent states.